ib_cni_stop.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+export LOGDIR=$WORKSPACE/logs
+export ARTIFACTS=$WORKSPACE/artifacts
+
+for ds in $(/usr/local/bin/kubectl -n kube-system get ds |grep kube|awk '{print $1}'); do
+    /usr/local/bin/kubectl -n kube-system delete ds $ds
+done
+
+for sc in $(screen -ls|grep multus|awk '{print $1}'); do
+    screen -X -S $sc quit
+done
+
+kill $(ps -ef |grep local-up-cluster.sh|grep $WORKSPACE|awk '{print $2}')
```

Comment:
why this is not in common?

Reply:
@moshe010 ,To keep the workflow as is (having three scripts, install, start, and test), if i want to split the stop script without impacting the workflow, we will need to change the CI job script to point to the common stop script instead of running {project}_cni_stop.sh, otherwise we will need to call it from the install script, which will cause the stop script to run every time we run the install script. if you like i can change the ci script to run a common script?

Reply:
so put comment that we need to fix this